mlir/test/Dialect/Rock/lowering_xdlops_gemm_v2.mlir

```diff
@@ -1,15 +1,20 @@
 // RUN: rocmlir-opt -rock-threadwise-gemm-lowering %s | FileCheck %s
+
 func.func @rock_xdlops_gemm_v2_reduction_nokpack(%matrixA : memref<2xf32, 5>,
                                                  %matrixB : memref<2xf32, 5>,
                                                  %matrixC : memref<2xvector<16xf32>, 5>) {
-  %c0 = arith.constant 0 : index
   // CHECK-LABEL: func.func @rock_xdlops_gemm_v2_reduction_nokpack
-  // CHECK: rock.in_bounds_load
-  // CHECK: rock.in_bounds_load
-  // CHECK: amdgpu.mfma
+  // CHECK-SAME: ([[ABuf:%.+]]: memref<2xf32, 5>, [[BBuf:%.+]]: memref<2xf32, 5>, [[CBuf:%.+]]: memref<2xvector<16xf32>, 5>)
+  // CHECK: rock.transforming_for
+  // CHECK-SAME: bounds [2]
+  // CHECK: [[a:%.+]] = memref.load [[ABuf]]
+  // CHECK: [[b:%.+]] = memref.load [[BBuf]]
+  // CHECK: [[c:%.+]] = memref.load [[CBuf]]
+  // CHECK: amdgpu.mfma [[a]] * [[b]] + [[c]] {{.*}} : f32, vector<16xf32>
+  %c0 = arith.constant 0 : index
   rock.xdlops_gemm_v2 %matrixC += %matrixA[%c0] * %matrixB[%c0] {
     params = #rock.xdlops_gemm_params<
-       kPerBlock = 8,
+       kPerBlock = 4,
```

Comment:
Why do you have to change this?

Reply:
I had to change this so the test would actually run correctly